oaknational/Oak-Web-Application
# Feat/lesq 354/card hover updates [LESQ-354]

## Description

- Updated background colour on hover for: `ListItemCard`, `ListItemIndexDesktop, `ListItemIconDesktop` and `ProgrammeListItem`
- Removed transform on hover from `ProgrammeListItem`, `ListItemCard` and `ListItemIndexDesktop` and removed drop shadow change
- Added an override for early release units as they use custom coloring
- Added tests for the colour changes

## How to test

1. Go to https://deploy-preview-2006--oak-web-application.netlify.thenational.academy/teachers/programmes/english-secondary-ks3/units
2. Hover over the unit
3. You should see the new hover state
4. Click through to lessons and hover

## Screenshots

How it used to look (delete if n/a):

<img width="500" alt="Screenshot 2023-10-11 at 09 16 08" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/8404749c-8a53-4e94-9eb2-c5ff31ce4781">


How it should now look:

<img width="500" alt="Screenshot 2023-10-11 at 09 15 45" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/4ea3e069-c37b-4d1e-b5dd-ea779f6dd992">


## ACs
- [ ]  Update hover state so the lesson/unit title gets underlined and add colour transition (a bit like in [[here](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_buttons_fade1)](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_buttons_fade1)) to the coloured box (blue one on the unit listing and pink on the lesson listing)
- [ ]  On hover make pink/blue background darker, change white background to a very light grey, see Figma for values, underline main heading only
- [ ]  Apply to unit listing card
- [ ]  Apply to search results card
- [ ]  Apply to lesson listing card
- [ ]  Apply to card on programmes page (the one with tiers)
